src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.tasks.Task;
+import duke.ui.Ui;
+
+import java.io.IOException;
```

Comment:
Maybe you could take a look at `Duke.java`? It is the same issue, and for the rest of the classes too! 😅